frontend/public/templates/work_packages/tabs/_add_work_package_relation.html

```diff
@@ -9,6 +9,7 @@
     id="relation_to_id-{{ handler.relationsId }}"
     name="relation[to_id][{{ handler.relationsId }}]"
     ng-model="$parent.relationToAddId"
+    title="{{ I18n.t('js.relation_button_inputs.' + handler.relationsId) }}"
```

Comment:
@myabc if the German translation is suitable and if we don't need some more eloquent text like "start typing for {{name}} task autocomplete"